q2_diversity/_ordination.py

```diff
@@ -5,3 +5,9 @@
 #
 # The full license is in the file LICENSE, distributed with this software.
 # ----------------------------------------------------------------------------
+
```

Comment:
Don't know why git is calling this a file rename - just deletion of one and creation of the other.